docs/docs/architecture/secrets.md

```diff
@@ -22,3 +22,16 @@ It needs to be provided with the secret seed, from which it can derive the signi
 This procedure is called recovery and is initiated by the workload owner.
 The CLI decrypts the secret seed using the private seed share owner key, verifies the Coordinator and sends the seed through the `Recover` method.
 The Coordinator recovers its key material and verifies the manifest history signature.
+
+## Workload Secrets
+
+The Coordinator provides each workload a secret seed during attestation. This secret can be used by the workload to derive additional secrets for example to
+encrypt persistent data. Like the workload certificates it's mounted in the shared Kubernetes volume `contrast-secrets` in the path `<mountpoint>/secrets/workload-secret`.
+
+:::warning
+
+The data owner shouldn't encrypt data with secrets derived from the workload secret if they want to keep the data private against the workload owner.
+The workload owner can derive the workload secret themself, since it's derived from the secret seed known to the workload owner.
```

Comment:
I think themself is correct since we also use they/them as the singular form as in academic writing. Also see: https://www.merriam-webster.com/wordplay/themself

Reply:
Ah interesting APA says it's also ok to use the themselves for singular they: https://apastyle.apa.org/style-grammar-guidelines/grammar/singular-they. Will go with you suggestion then.